# system param
DOB_rate: 0.0



# Desired positon for pos_control HOLD
POS_X: 0.0
POS_Y: 0.0
POS_Z: 1.0

# PID parameters for pos_error
MC_X_P: 3.0
MC_X_I: 0.05
MC_X_D: 0.0

MC_Y_P: 3.0
MC_Y_I: 0.05
MC_Y_D: 0.0

MC_Z_P: 1.5
MC_Z_I: 0.05
MC_Z_D: 0.0

# PID parameters for vel_error
MC_VX_P: 3.0
MC_VX_I: 0.3
MC_VX_D: 0.5

MC_VY_P: 3.0
MC_VY_I: 0.3
MC_VY_D: 0.5

MC_VZ_P: 1
MC_VZ_I: 0.2
MC_VZ_D: 0.0